tests/homestead/test_state_transition.py

```diff
@@ -114,3 +115,22 @@ def test_general_state_tests(test_file: str) -> None:
     except KeyError:
         # KeyError is raised when a test_file has no tests for homestead
         raise pytest.skip(f"{test_file} has no tests for homestead")
+
+
+# Test Invalid Block Headers
+run_invalid_header_test = partial(
+    run_homestead_blockchain_st_tests,
+    "tests/fixtures/LegacyTests/Constantinople/BlockchainTests/InvalidBlocks/bcInvalidHeaderTest",
+)
+
+
+@pytest.mark.parametrize(
+    "test_file_parent_hash",
+    [
+        "wrongParentHash.json",
+        "wrongParentHash.json",
```

Comment:
`"wrongParentHash.json"` is repeated twice here. I think it should `"wrongParentHash2.json"`

Reply:
Corrected. Thanks!